src/views/pages/Diagnostics/index.js

```diff
@@ -0,0 +1,55 @@
+import React, { useState, useEffect } from "react";
+import Node from '../../../services/Node';
+
+function Diagnostics() {
+
+    const [data, setData] = useState({ pending: true, diagnostics: null });
+
+    useEffect(() => {
+        setData({ pending: true, diagnostics: {} });
+
+        const fetchData = async () => {
+            const node = new Node();
```

Comment:
This reminds me that we need to figure out how we want to organize the "registration" of services. I think every service will end up being a singleton and be instantiated at the root app level. There is a nice way of doing this. Regardless, this is fine for now.